src/Curators/Publisher.php

```diff
@@ -20,24 +14,19 @@ class Publisher
     private function __construct(string $name)
     {
         if (!self::isValid($name)) {
-            throw new InvalidArgumentException('Unknown publisher: ' . $name);
+            throw new InvalidArgumentException('Invalid publisher: ' . $name);
         }
         $this->name = $name;
     }
 
-    public static function bruzz(): self
-    {
-        return new self(self::BRUZZ);
-    }
-
     public static function fromName(string $name): self
     {
         return new self($name);
     }
 
     private static function isValid(string $name): bool
     {
-        return in_array($name, self::$knownPublishers);
+        return !empty($name);
     }
 
     public function getName(): string
```

Comment:
Similarly I'd expect the VO to represent the name, so `getName()` is again a bit unclear. `toString()` might be better?